_data/package.yml

```diff
@@ -51,7 +51,7 @@ needed-pkgs:
       url: "https://gap-packages.github.io/io"
     - name: "digraphs"
       version: ">=0.12.0"
-      url: "https://gap-packages.github.io/Digraphs"
+      url: "https://gap-packages.github.io/digraphs"
```

Comment:
This gives a 404 without the capital letter. How can we avoid this happening?

Reply:
This can be changed manually, and we should!  I'll add a commit now...

Reply:
(to be clear, this was my mistake and not the result of a script)